sapi/phpdbg/xml.md

```diff
@@ -558,7 +558,7 @@ set
 - generally enables / disables quietness silently with no further xml answer
 - if the boolean switch is omitted, it emits current state in a &lt;setquiet active="" /> where active is on or off
 
-### setpping ###
+### stepping ###
 
 - sets stepping to either opcode or line (so a step command will either advance one op or one line)
```

Comment:
I'm not sure how to reconcile this w/ the previous, but it might be the right change